test/helpers/chains.js

```diff
@@ -0,0 +1,136 @@
+// The following listing does not pretend to be exhaustive or even accurate. It SHOULD NOT be used in production.
+
+const { ethers } = require('hardhat');
+const { mapValues } = require('@openzeppelin/contracts/test/helpers/iterate');
+
+// EVM (https://axelarscan.io/resources/chains?type=evm)
+const ethereum = {
+  Ethereum: 1n,
+  optimism: 10n,
+  binance: 56n,
+  Polygon: 137n,
+  Fantom: 250n,
+  fraxtal: 252n,
+  filecoin: 314n,
+  Moonbeam: 1284n,
+  centrifuge: 2031n,
+  kava: 2222n,
+  mantle: 5000n,
+  base: 8453n,
+  immutable: 13371n,
+  arbitrum: 42161n,
+  celo: 42220n,
+  Avalanche: 43114n,
+  linea: 59144n,
+  blast: 81457n,
+  scroll: 534352n,
+  aurora: 1313161554n,
+};
+
+const solana = {
+  Mainnet: '5eykt4UsFv8P8NJdTREpY1vzqKqZKvdpKuc147dw2N9d',
+};
+
+const CAIP350 = {
+  eip155: { chainType: '0x0000', encoder: { reference: x => ethers.toBigInt(x).toString(), address: ethers.hexlify } },
+  solana: { chainType: '0x0002', encoder: { reference: ethers.encodeBase58, address: ethers.encodeBase58 } },
+};
+
+const asHex = value => {
+  if (value === undefined || value === null || value === '') return '0x';
+  try {
+    return ethers.toBeHex(value);
+  } catch {
+    /* Do nothing */
+  }
+  try {
+    return ethers.toBeHex(ethers.decodeBase58(value));
+  } catch {
+    /* Do nothing */
+  }
+  try {
+    return ethers.toBeHex(ethers.decodeBase64(value));
+  } catch {
+    /* Do nothing */
+  }
+  throw new Error(`Unable to decode ${value}`);
+};
+
+const formatERC7913v1 = ({ type, reference, address }) => {
```

Comment:
Shouldn't it be ERC7930?
```suggestion
const formatERC7930v1 = ({ type, reference, address }) => {
```